dataeng/jobs/analytics/EnterpriseEnrollment.groovy

```diff
@@ -0,0 +1,32 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
+
+class EnterpriseEnrollment {
+    public static def job = { dslFactory, allVars ->
+        dslFactory.job("enterprise-enrollment") {
+            authorization {
+                allVars.get('PROJECT_BASED_ACCESS').each { roles ->
+                    permission(roles.get('ROLE'), roles.get('GITHUB_ID'))
```

Comment:
Oh, right.   We talked about changing this to be a double nested list.    That is, loop through IDs, and then loop through roles per id.